app/controllers/phrasing_phrases_controller.rb

```diff
@@ -6,7 +6,7 @@ class PhrasingPhrasesController < Phrasing.parent_controller.constantize
 
   include PhrasingHelper
 
-  before_filter :authorize_editor
+  before_action :authorize_editor
```

Comment:
Rails 5 finally dropped support for `*_filter` methods. I will modify the `.travis.yml` file if it's ok to drop Rails 3.2.0 support.